pkgs/development/lua-modules/overrides.nix

```diff
@@ -711,6 +711,8 @@ in
     };
   });
 
+  lux-lua = final.callPackage ./lux-lua.nix { inherit lua; };
+
```

Comment:
Thinking we should just move this to a top-level `pkgs/by-name` since it's just a rust application, really.